gulp/typescript.js

```diff
@@ -53,4 +55,17 @@ module.exports = (gulp, plugins, blueprint) => {
             tsResult.dts,
         ]).pipe(blueprint.dest(project));
     });
+
+    const bundleTaskNames = blueprint.projectsWithBlock("typescript").map((project) => {
+        const taskName = `typescript-bundle-${project.id}`;
```

Comment:
Not really, because `generateWebpackBundleConfig` uses `dist/index.js` as an entry point (not a TS file). This is the whole deal with splitting up our build process into a `tsc` stage and a webpack stage. We can follow up further offline about this if you want.